config/locales/en/course/mailer/user_invitation_email.yml

```diff
@@ -7,5 +7,7 @@ en:
           You are invited to register for %{course} on %{coursemology}.
 
 
-          You do not seem to have an account registered at this email address; please %{sign_up}
-          for an account, and register for %{course} using the following registration code:
+          You do not seem to have an account registered at this email address (%{email});
+          please %{sign_up} for an account before visiting %{course} to enrol for the course.
+          If you have an existing Coursemology account registered under a different email,
+          enter the following personal registration code when you enrol:
```

Comment:
Shall we still mention should not share code with others? 

Next step we can make this a link, user clicks the link, it goes to the course show page and the code fills in the box automatically :-P